readme.md

```diff
@@ -6,6 +6,7 @@
 
 - [Ruby 2.6.x](https://www.ruby-lang.org/en/documentation/installation)
 - [Golang](https://golang.org/doc/install)
+- [Node.js >=12.19.0](https://nodejs.org/en/)
```

Comment:
Hey thanks for the PR @Levash0v , just wondering why you think nodejs is a requirement? So far this isnt a requirement to get a sifnode up and running at all? Did you get an issue of sorts at all?

Reply:
When installing Starport locally, the following packages are required:
_Prerequisites:_ If you want to install Starport locally, make sure to have Golang> = 1.14 and Node.js> = 12.19.0 installed on your system._
During the initial setup, I installed two packages Noda. js and Golang (Ruby did not install it, this requirement appeared later) the node started, later installed the Ruby package

Reply:
yeah ok I get it now. We only used starport to scaffold the app and arent really using it now we have the repo under development. We also cut out the frontend vue code that came with the scaffolder and are building out our own frontend foundations. ill have to get ride of the referrence to starport in the doco now and its really not a requirement any more.